src/models/db.js

```diff
@@ -27,7 +27,16 @@ let deferreds = [];
 let state = IDLE;
 let db    = null;
 
-module.exports = () => {
+const { name, user, password, host, port } = config.get('db');
+
+const sequelize = new Sequelize(name, user, password, {
+  host,
+  port,
+  dialect: 'postgres',
+  logging: false
+});
```

Comment:
extracting this out of the exported function makes it possible to export it so that we can use it elsewhere. (in my case: for the session store)